### PR TITLE
4.3.7 SP5 patch jcayouette (#2381)

### PR DESCRIPTION
**Pull request by keichwa**
# Description

SLE 15 SP4 as base system for SUMA 4.3.8.

https://github.com/SUSE/spacewalk/issues/22684

# Target branches

Which documentation version does this PR apply to?
- manager-4.3.8

# Links
- Related development PR #<insert PR link, if any>
- Fixes #<insert spacewalk issue, if any>

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- 4.3 based on SLE 15 SP5
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- SUSE Manager 4.3 is based on SUSE Linux Enterprise 15 SP5; changed installation
  procedure accordingly
```

**Review by raulillo82 — CHANGES_REQUESTED** [review]
I might've found an error or at least an unfortunate typo.

**Comment by keichwa** [comment]
Grrh, yes, this was a super stupid typo ;-(  I'll check the other 2...

File: modules/installation-and-upgrade/pages/proxy-registration.adoc [hunk 2]
```diff
@@ -33,10 +33,10 @@ image::proxy-key.png[]
...
 . From the [guimenu]``Child Channels`` listing select the recommended channels by clicking the ``include recommended`` icon:
 +
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-SP4-Pool
+* SLE-Module-Basesystem15-SP4-Updates
+* SLE-Module-Server-Applications15-SP4-Pool
+* SLE-Module-Server-Applications15-SP4-Updates
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/proxy-registration.adoc:39]:
```suggestion
* SLE-Module-Basesystem15-{sp-base}-Pool
* SLE-Module-Basesystem15-{sp-base}-Updates
* SLE-Module-Server-Applications15-{sp-base}-Pool
* SLE-Module-Server-Applications15-{sp-base}-Updates
```

File: modules/quickstart/pages/quickstart-install-suma-proxy.adoc [hunk 3]
```diff
@@ -61,10 +61,10 @@ image::proxy-key.png[]
...
 . From the [guimenu]``Child Channels`` listing select the recommended channels by clicking the ``include recommended`` icon:
 +
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-SP4-Pool
+* SLE-Module-Basesystem15-SP4-Updates
+* SLE-Module-Server-Applications15-SP4-Pool
+* SLE-Module-Server-Applications15-SP4-Updates
```
Comment (keichwa) [inline modules/quickstart/pages/quickstart-install-suma-proxy.adoc:67]:
```suggestion
* SLE-Module-Basesystem15-{sp-base}-Pool
* SLE-Module-Basesystem15-{sp-base}-Updates
* SLE-Module-Server-Applications15-{sp-base}-Pool
* SLE-Module-Server-Applications15-{sp-base}-Updates
```

File: modules/installation-and-upgrade/pages/proxy-registration.adoc [hunk 4]
```diff
@@ -33,10 +33,10 @@ image::proxy-key.png[]
...
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
 * SLE-Module-SUSE-Manager-Proxy-4.3-Pool
 * SLE-Module-SUSE-Manager-Proxy-4.3-Updates
```
Comment (deneb-alpha) [inline modules/installation-and-upgrade/pages/proxy-registration.adoc:41]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Updates
```

File: modules/quickstart/pages/quickstart-install-suma-proxy.adoc [hunk 5]
```diff
@@ -61,10 +61,10 @@ image::proxy-key.png[]
...
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
 * SLE-Module-SUSE-Manager-Proxy-4.3-Pool
 * SLE-Module-SUSE-Manager-Proxy-4.3-Updates
```
Comment (deneb-alpha) [inline modules/quickstart/pages/quickstart-install-suma-proxy.adoc:69]:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Updates
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
left one suggestion about using  `{productnumber}` instead  of hardcoded `4.3`

**Review by deneb-alpha — APPROVED** [review]
LGTM, thanks

**Comment by deneb-alpha** [comment]
> left one suggestion about using `{productnumber}` instead of hardcoded `4.3`

as discussed. this will be done in a different card, not now

**Review by jcayouette — APPROVED** [review]

**Review by raulillo82 — APPROVED** [review]

**Pull request merged**